composer.json

```diff
@@ -4,45 +4,45 @@
     "description": "AppMerchant module",
     "license": "proprietary",
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.2",
+        "spryker/app-kernel": "^1.0.0",
+        "spryker/event-dispatcher": "^1.6.0",
         "spryker/glue-application-extension": "^1.0.0",
+        "spryker/glue-backend-api-application": "*",
+        "spryker/glue-json-api-convention": "^1.3.0",
+        "spryker/http": "^1.11.0",
         "spryker/kernel": "^3.30.0",
         "spryker/log": "^3.0.0",
         "spryker/message-broker": "^1.11.0",
-        "spryker/transfer": "^3.33.0"
+        "spryker/message-broker-aws": "*",
+        "spryker/propel": "*",
```

Comment:
`*` usually have to be changed to a minimal possible version.

Reply:
Missed it, updated.